test/specs/components/Popup/Popup-test.tsx

```diff
@@ -131,4 +184,11 @@ describe('Popup', () => {
       expect(spy.mock.calls[0][1]).toMatchObject({ open: true })
     })
   })
+
+  describe('open/close popup by keyboard', () => {
+    togglePopupTest('enter', keyboardKey.Enter, 'click')
```

Comment:
nit: lets rename it in a way `togglePopupTest ` -> `testPopupToggleOn`, this will make test's code more readable